src/utils/dates.js

```diff
@@ -80,8 +80,8 @@ let dates = {
     return dates.eq(dateA, dateB, 'month')
   },
 
-  isToday(date) {
-    return dates.eq(date, dates.today(), 'day')
+  sameDay(dateA, dateB) {
+    return dates.eq(dateA, dateB, 'day')
```

Comment:
i think maybe this is doesn't add a ton of value over using `dates.eq` each time, i'd remove the helper personally

Reply:
was copying the `sameMonth` example just above. I agree - will remove. I thought it wise to remove `isToday` (no longer used) since it encourages bad (future) behavior. Agree?

Reply:
👍